layouts/partials/footer.html

```diff
@@ -115,4 +115,14 @@ <h2 class="section-title">{{ i18n "footer-other-section-text" }}</h2>
     });
   {{ end }}
 })(jQuery, document);
+
+jQuery(document).ready(function($){ 
```

Comment:
I assume this is no longuer needed now that we decided to drop the selection input option?

Reply:
forgot to remove it, now removed.